src/ophyd_async/fastcs/jungfrau/_utils.py

```diff
@@ -16,6 +19,7 @@ def create_jungfrau_external_triggering_info(
     Args:
         total_triggers: Total external triggers expected before ending acquisition.
         exposure_time_s: How long to expose the detector for each of its frames.
+        deadtime_s: Time between exposures
```

Comment:
Should: I feel like deadtime doesn't really make sense to a user in the case we're using it here. I get that we're discussing further but in the meantime I would either:
1. Not let the user specify it in `create_jungfrau_external_triggering_info` at all
2. Expand this docstring to be more detailed about why it's needed